src/languages/ux_markup.coffee

```diff
@@ -2,7 +2,7 @@ module.exports = {
 
   name: "UX Markup"
   namespace: "ux"
-  fallback: ['html']
+  fallback: ['xml']
```

Comment:
I just figured this grammar was more closely related to XML than HTML.  It did not appear to have any effect in the build.